src/encoding.ts

```diff
@@ -59,6 +59,26 @@ export interface Encoding<F> {
   // TODO: Ham need to add default behavior
   y2?: FieldDefWithoutScale<F> | ValueDef;
 
+  /**
+   * Error value of x coordinates for error specified `"errorbar"` and `"errorband"`.
+   */
+  xError?: FieldDefWithoutScale<F> | ValueDef;
```

Comment:
Are we okay with this being suggested as possible channel for other marks?

Reply:
I and @domoritz agreed that we would want to get this work first, then make `x/yError` only supported by error bar and error band later because it seems to be a big typing refactor in Vega-Lite.

Reply:
It might make sense to already have a new spec type for composite marks but I stand with my decision not to make new types for all mark types in the same PR as supporting errors.